dmscripts/notify_suppliers_of_awarded_briefs.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime, timedelta
+
+from dmutils.email.exceptions import EmailError
+from dmutils.email.helpers import hash_string
+from dmutils.formats import DATE_FORMAT
+from dmscripts.helpers import env_helpers
+
+
+def create_context_for_brief(stage, brief):
+    return {
+        'brief_title': brief['title'],
+        'brief_link': '{}/{}/opportunities/{}'.format(
+            env_helpers.get_web_url_from_stage(stage),
+            brief['framework']['family'],
+            brief['id']
+        )
+    }
+
+
+def main(
```

Comment:
I'd still go ahead and base the name of the function on what it does, even if that means giving it the same name as the module. Consider this - to use this, someone's probably going to be doing a `from dmscripts.notify_suppliers_of_awarded_briefs import main` - and then someone's going to have this annoyingly meaningless `main` floating around in their namespace...

Reply:
(two can play at that game)